src/main/java/org/apache/sling/feature/cpconverter/accesscontrol/DefaultAclManager.java

```diff
@@ -105,34 +100,15 @@ public void addRepoinitExtension(@NotNull List<VaultPackageAssembler> packageAss
             // system users
 
             for (SystemUser systemUser : systemUsers) {
-                // TODO does it harm?!?
-                addSystemUserPath(formatter, systemUser.getIntermediatePath());
-
                 // make sure all users are created first
-
                 formatter.format("create service user %s with path %s%n", systemUser.getId(), systemUser.getIntermediatePath());
-
-                // clean the unneeded ACLs, see SLING-8561
```

Comment:
i think it's ok.... but at a first glance i got confused by the removal of this line...... because what it refers to rather falls into SLING-9953.